### PR TITLE
Refactor login money transfer

### PR DESCRIPTION
**Pull request by DavidVallejosT**
Refactor login into Money Transfer project
Solve merge conflicts with new Code merged to Upstream/Master
Update Login logic to the new scenario

### CODE REVIEW

File: IDT.MoneyTransfer/Main/UI/Pages/Components/PortalWeb.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿
+namespace IDT.MoneyTransfer.Main.UI.Pages.Components
```
Comment (mabalderramav) [inline IDT.MoneyTransfer/Main/UI/Pages/Components/PortalWeb.cs]: Add comments

File: IDT.MoneyTransfer/Main/UI/Pages/Components/PortalWeb.cs [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+{
+    public enum PortalWeb
```
Comment (mabalderramav) [inline IDT.MoneyTransfer/Main/UI/Pages/Components/PortalWeb.cs]: Add comments

File: IDT.MoneyTransfer/Main/UI/Pages/Login/Components/UserType.cs [hunk 3]
```diff
@@ -0,0 +1,8 @@
+﻿namespace IDT.MoneyTransfer.Main.UI.Pages.Login.Components
```
Comment (mabalderramav) [inline IDT.MoneyTransfer/Main/UI/Pages/Login/Components/UserType.cs]: Add comments

File: IDT.MoneyTransfer/Main/UI/Pages/Login/Components/UserType.cs [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+{
+    public enum UserType
```
Comment (mabalderramav) [inline IDT.MoneyTransfer/Main/UI/Pages/Login/Components/UserType.cs]: Add comments

File: IDT.MoneyTransfer/Main/UI/Pages/MoneyTransfer/PayerLocationLookup/PayerLocationPage.cs [hunk 5]
```diff
@@ -0,0 +1,12 @@
+﻿// --------------------------------------------------------------------
+// <copyright file="PayerLocationPage.cs" company="IDT.MoneyTransfer">
+// Copyright (c) IDT.MoneyTransfer. All rights reserved.
+// </copyright>
+// -------------------------------------------------------------------
+
+namespace IDT.MoneyTransfer.Main.UI.Pages.MoneyTransfer.PayerLocationLookup
+{
+    public class PayerLocationPage
```
Comment (mabalderramav) [inline IDT.MoneyTransfer/Main/UI/Pages/MoneyTransfer/PayerLocationLookup/PayerLocationPage.cs]: If this class isn't using. you could remove it.

File: IDT.MoneyTransfer/Main/UI/Pages/Components/PortalWeb.cs (hunk 1 above)
Comment (DavidVallejosT) [reply IDT.MoneyTransfer/Main/UI/Pages/Components/PortalWeb.cs]: Done.

File: IDT.MoneyTransfer/Main/UI/Pages/Components/PortalWeb.cs (hunk 2 above)
Comment (DavidVallejosT) [reply IDT.MoneyTransfer/Main/UI/Pages/Components/PortalWeb.cs]: Done.

File: IDT.MoneyTransfer/Main/UI/Pages/Login/Components/UserType.cs (hunk 3 above)
Comment (DavidVallejosT) [reply IDT.MoneyTransfer/Main/UI/Pages/Login/Components/UserType.cs]: Done.

File: IDT.MoneyTransfer/Main/UI/Pages/Login/Components/UserType.cs (hunk 4 above)
Comment (DavidVallejosT) [reply IDT.MoneyTransfer/Main/UI/Pages/Login/Components/UserType.cs]: Done.

File: IDT.MoneyTransfer/Main/UI/Pages/MoneyTransfer/PayerLocationLookup/PayerLocationPage.cs (hunk 5 above)
Comment (DavidVallejosT) [reply IDT.MoneyTransfer/Main/UI/Pages/MoneyTransfer/PayerLocationLookup/PayerLocationPage.cs]: Done.

**Pull request merged**